docs/sql/index.md

```diff
@@ -56,8 +56,375 @@ When you return data in CSV or raw format, each row corresponds to a *document*,
 
 ## User interfaces
 
-You can test queries using **Dev Tools** in Kibana (`https://<host>:5601`).
+Kibana
+{: .label .label-yellow :}
 
+### Index data
+
+The SQL plugin is for read-only purposes, so you cannot index or update data using SQL.
+
+Use the `bulk` operation to index some sample data:
+
+```json
+PUT accounts/_bulk?refresh
+{"index":{"_id":"1"}}
+{"account_number":1,"balance":39225,"firstname":"Amber","lastname":"Duke","age":32,"gender":"M","address":"880 Holmes Lane","employer":"Pyrami","email":"amberduke@pyrami.com","city":"Brogan","state":"IL"}
+{"index":{"_id":"6"}}
+{"account_number":6,"balance":5686,"firstname":"Hattie","lastname":"Bond","age":36,"gender":"M","address":"671 Bristol Street","employer":"Netagy","email":"hattiebond@netagy.com","city":"Dante","state":"TN"}
+{"index":{"_id":"13"}}
+{"account_number":13,"balance":32838,"firstname":"Nanette","lastname":"Bates","age":28,"gender":"F","address":"789 Madison Street","employer":"Quility","email":"nanettebates@quility.com","city":"Nogal","state":"VA"}
+{"index":{"_id":"18"}}
+{"account_number":18,"balance":4180,"firstname":"Dale","lastname":"Adams","age":33,"gender":"M","address":"467 Hutchinson Court","email":"daleadams@boink.com","city":"Orick","state":"MD"}
+```
+
+Here’s how core SQL concepts map to Elasticsearch:
+
+| SQL | Elasticsearch | Example
+:--- | :--- | :---
+Table | Index | `accounts`
+Row | Document | `1`
+Column | Field | `account_number`
+
+To list all your indices:
+
+```sql
+SHOW TABLES LIKE %
+```
+
+| id | TABLE_NAME
+:--- | :---
+0 | accounts
+
+### Read data
+
+After you index a document, retrieve it using the following SQL expression:
+
+```sql
+SELECT *
+FROM accounts
+WHERE _id = 0
```

Comment:
Am I missing something, or is Amber indexed above with an ID of 1 in your _bulk statement?

Reply:
Yes, changed "_id = 1".